site/en/blog/chromium-chronicle-20/index.md

```diff
@@ -0,0 +1,93 @@
+---
+title: "The Chromium Chronicle #20: Benchmarking Test Harnesses"
```

Comment:
Is "#20" consistent with the other editions? (If this were the first edition, I recommend "1" instead of "#1".)

Reply:
Yep, it's consistent.